src/app/space/settings/areas/create-area-dialog/create-area-dialog.component.ts

```diff
@@ -37,6 +37,10 @@ export class CreateAreaDialogComponent {
     this.nameInput.nativeElement.focus();
   }
 
+  public clearField() {
```

Comment:
No need for the "public" modifier here and we generally don't include it in the codebase (at least Deployments area).

Reply:
Also `: void` return please